common/src/main/java/io/druid/collections/BlockingPool.java

```diff
@@ -77,15 +77,16 @@ public int getPoolSize()
    * Take a resource from the pool, waiting up to the
    * specified wait time if necessary for an element to become available.
    *
-   * @param timeout maximum time to wait for a resource, in milliseconds.
+   * @param timeoutMs maximum time to wait for a resource, in milliseconds.
    *
    * @return a resource, or null if the timeout was reached
    */
-  public ReferenceCountingResourceHolder<T> take(final long timeout)
+  public ReferenceCountingResourceHolder<T> take(final long timeoutMs)
```

Comment:
I don't understand this comment.. If timeoutMs = 0, then take() immediately returns an object or null without waiting at all. Actually, negative ```timeoutMs```s have the same meaning.